functional-tests/helpers/cleanup.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable func-names */
```

Comment:
AFAIK you can bundle all of your `eslint-disables` into one statement.

Reply:
👍